kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
@kartben @ctron @dwoodard1 @MMaiero I agree with @dwoodard1 on the minor version. Indeed I changed it this commit : https://github.com/eclipse/kura/pull/1110/commits/5df8404debc49a73d0f166d596bfa08786751ac4
In general, I avoid to change the API, but in this case the only way to easily fix the issue was to add a new method and deprecate the old one. Moreover, it seems to me that v3.0.0 release is a good moment to add something to the API.